docs/dataservice-write-example.md

```diff
@@ -34,6 +34,39 @@ To execute the example, run:
 
 If everything is fine, you would see `Publish Successful - TraceID: <TraceId generated by the platform>`.
 
+### Building on Android
```

Comment:
should be a h2.
## Building and running on Linux

## Building and running on Android

Reply:
Done